lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require_relative "menu"
+
+class Order
+
+  def initialize(menu = Menu.new)
+    @menu = menu.list
+    @item = nil
+    @quantity = nil
+    @basket = []
+    @ordering = true
+  end
+
+  def choose
+    loop_select
+  end
+
+  private
+
+  def loop_select
+    while @ordering do
```

Comment:
Why use a loop and not a method?

Reply:
I am using a loop which call all the other methods many times to offer the possibility to the user to continue to order as much dishes as he wants until the user clearly say that he does not wish to order anything else